src/components/common/DashboardSidebar.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 import DashboardSidebarStyled from './DashboardSidebarStyled';
 import codeClanLogo from '../assets/image/codeClanLogoWhite.png';
+import { Popconfirm } from 'antd';
+import { QuestionCircleOutlined, LogoutOutlined } from '@ant-design/icons';
+import { Link, Redirect } from 'react-router-dom';
```

Comment:
Codacy found an issue: ['Redirect' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5803328)